app/styles/components/nav-bar.css

```diff
@@ -0,0 +1,105 @@
+.nav-background {
+    background-image: url("/assets/images/nav-background.png");
+    height: 180px;
+    position: relative;
+}
+
+.nav-div {
+    bottom: 0;
+    position: absolute;
+    width: 100%;
+}
+
+nav {
+    font-size: 0;
+    height: 50px;
+	position: relative;
+}
+
+nav a {
+    color: white;
+    cursor: pointer;
+    display: inline-block;
+	font-size: 15px;
+	height: 100%;
+	line-height: 50px;
+	position: relative;
+	text-decoration: none;
+	text-transform: uppercase;
+    text-align: center;
+    z-index: 1;
+}
+
+nav .animation {
+    height: 100%;
+	position: absolute;
+    top: 0;
+    transition: all 0.5s ease 0s;
+	z-index: 0;
+}
+
+a:nth-child(1) {
+	width: 120px;
+}
+
+a:nth-child(2) {
+	width: 120px;
+}
+
+a:nth-child(3) {
+	width: 120px;
+}
+
+a:nth-child(4) {
+	width: 120px;
+}
+
+a:nth-child(5) {
+	width: 120px;
+}
+
+a:nth-child(6) {
+	width: 120px;
+}
+
+nav a:nth-child(1):hover~.animation {
+	background-color: lightgray;
+	left: 0;
+	opacity: 0.4;
+	width: 120px;
+}
+
+nav a:nth-child(2):hover~.animation {
+	background-color: lightgray;
+	left: 120px;
+	opacity: 0.4;
+	width: 120px;
+}
+
+nav a:nth-child(3):hover~.animation {
+	background-color: lightgray;
+	left: 240px;
+	opacity: 0.4;
+	width: 120px;
+}
+
+nav a:nth-child(4):hover~.animation {
+	background-color: lightgray;
+	left: 360px;
+	opacity: 0.4;
+	width: 120px;
+}
+
+nav a:nth-child(5):hover~.animation {
+	background-color: lightgray;
+	left: 480px;
+	opacity: 0.4;
+	width: 120px;
+}
+
+nav a:nth-child(6):hover~.animation {
+	background-color: lightgray;
+	left: 600px;
+	opacity: 0.4;
```

Comment:
Please use .scss styles instead, check their possibilities and use them to reduce duplicated styles here

Reply:
I will gradually replace .css style files